### ISSUE TITLE
Issues with Mini-NDN installation

### ISSUE THREAD

**Issue by dulalsaurab** [named-data/mini-ndn#69]
Can't install Mini-NDN due to the following issue. 

```
.../ndn-src/mini-ndn/dl/mininet-wifi /home/map901/mguard/ndn-src/mini-ndn/dl/mininet-wifi
fatal: unsafe repository ('/home/map901/mguard/ndn-src/mini-ndn/dl/mininet-wifi' is owned by someone else)
To add an exception for this directory, call:

	git config --global --add safe.directory /home/map901/mguard/ndn-src/mini-ndn/dl/mininet-wifi
```
running `git config --global --add safe.directory /home/map901/mguard/ndn-src/mini-ndn/dl/mininet-wifi` this command doesn't solves the problem.

edit:
running the above command (`sudo git config --global -- ..`)  with `sudo` resolved the problem.

**Comment by cawka** [comment]
You better chown repositories to the same user and run as that one

**Comment by yoursunny** [comment]
ndn-cxx has a similar bug https://redmine.named-data.net/issues/5205 but the error is not fatal.

**Comment by yoursunny** [comment]
This is fixed in 1e720875d2af06075a077a9d8bd6144ba33b2741
